index.js

```diff
@@ -1,55 +1,65 @@
-const github = require("@actions/github");
-const core = require("@actions/core");
-
-async function run() {
-  const inputs = {
-    githubToken: core.getInput("github_token", { required: true }),
-    prTitle: core.getInput("pr_title"),
-    prBody: core.getInput("pr_body"),
-    destinationBranch: core.getInput("destination_branch")
-  };
+const github = require('@actions/github');
+const core = require('@actions/core');
 
-  const base = inputs.destinationBranch;
-  const source = github.context.ref.replace(/^refs\/heads\//, "");
+const run = async () => {
+  const githubToken = core.getInput('github_token', { required: true });
+  const prTitle = core.getInput('pr_title');
+  const prBody = core.getInput('pr_body');
+  const baseBranch = core.getInput('destination_branch');
+  const sourceBranch = github.context.ref.replace(/^refs\/heads\//, '');
 
-  const octokit = github.getOctokit(inputs.githubToken);
-  core.info(`Look up a pull request with source=${source} base=${base}`);
-  const { data: pulls } = await octokit.rest.pulls.list({
+  const credentials = {
     owner: github.context.repo.owner,
     repo: github.context.repo.repo,
-    base,
-    head: `${github.context.repo.owner}:${source}`
+  };
+
+  const octokit = github.getOctokit(githubToken);
+  core.info(`Looking up a pull request with source branch="${sourceBranch || '<not found>'}" and base branch="${baseBranch || '<not specified>'}"`);
+
+  const branchHead = `${credentials.owner}:${sourceBranch}`;
+  const { data: pulls } = await octokit.rest.pulls.list({
+    ...credentials,
+    base: baseBranch,
+    head: branchHead,
   });
 
-  if (pulls.length == 0) {
-    core.info(`No such pull request: source=${source} base=${base}`);
-    return;
+  if (pulls.length === 0) {
+    throw new Error(`No pull request found for source="${source || '<not found>'}" base=${baseBranch || '<not specified>'}`);
```

Comment:
`throw` would stop the script execution here and would be reported to the `core`